src/main/scala/peregin/gpv/format/GpsFormatter.scala

```diff
@@ -0,0 +1,39 @@
+package peregin.gpv.format
+
+
+object GpsFormatter {
+
+  def formatLon(lon: Double): String = {
+    return formatAxis("WE", lon)
```

Comment:
minor nitpicking, in `Scala` is not necessary to add the `return` keyword, the return value is always the evaluation of the last expression or value (is used explicitly for early returns).
e.g. this is usually expressed as `def formatLon(lon: Double): String = formatAxis("WE", lon)`